::::::::::::::::::::
/usr/WS1/han12/.jacamar-ci/builds/4wyMqbxi/000/gitlab/han12/GEOSX/build/bin/geosx -i co2_thermal_2d.xml -x 2 -y 2 -z 1 -n 0to11 -o /usr/WS1/han12/.jacamar-ci/builds/4wyMqbxi/000/gitlab/han12/GEOSX/integratedTests/update/run/thermalMultiphaseFlow/co2_thermal_2d_04 --suppress-move-logging
::::::::::::::::::::
::::::::::::::::::::
/usr/tce/packages/python/python-2.7.16/bin/python -m mpi4py /usr/WS1/han12/.jacamar-ci/builds/4wyMqbxi/000/gitlab/han12/GEOSX/integratedTests/geosxats/helpers/restartcheck.py -a 1e-06 -r 1e-05 -w /usr/WS1/han12/.jacamar-ci/builds/4wyMqbxi/000/gitlab/han12/GEOSX/integratedTests/update/run/thermalMultiphaseFlow/co2_thermal_2d_04/0to11_restart_[0-9]+\.root /usr/WS1/han12/.jacamar-ci/builds/4wyMqbxi/000/gitlab/han12/GEOSX/integratedTests/update/run/thermalMultiphaseFlow/baselines/co2_thermal_2d_04/0to11_restart_[0-9]+\.root
::::::::::::::::::::
srun: Job 9479475 step creation temporarily disabled, retrying (Requested nodes are busy)
srun: Job 9479475 step creation still disabled, retrying (Requested nodes are busy)
srun: Step created for job 9479475
2: /usr/WS1/han12/.jacamar-ci/builds/4wyMqbxi/000/gitlab/han12/GEOSX/integratedTests/geosxats/helpers/restartcheck.py:246: RuntimeWarning: invalid value encountered in divide
2:   relative_difference = difference / abs_base_arr
0: /usr/WS1/han12/.jacamar-ci/builds/4wyMqbxi/000/gitlab/han12/GEOSX/integratedTests/geosxats/helpers/restartcheck.py:246: RuntimeWarning: invalid value encountered in divide
0:   relative_difference = difference / abs_base_arr
3: /usr/WS1/han12/.jacamar-ci/builds/4wyMqbxi/000/gitlab/han12/GEOSX/integratedTests/geosxats/helpers/restartcheck.py:246: RuntimeWarning: invalid value encountered in divide
3:   relative_difference = difference / abs_base_arr
1: /usr/WS1/han12/.jacamar-ci/builds/4wyMqbxi/000/gitlab/han12/GEOSX/integratedTests/geosxats/helpers/restartcheck.py:246: RuntimeWarning: invalid value encountered in divide
1:   relative_difference = difference / abs_base_arr
::::::::::::::::::::
/usr/WS1/han12/.jacamar-ci/builds/4wyMqbxi/000/gitlab/han12/GEOSX/build/bin/geosx -i co2_thermal_2d.xml -r co2_thermal_2d_04/0to11_restart_000000009 -x 2 -y 2 -z 1 -n 9to11 -o /usr/WS1/han12/.jacamar-ci/builds/4wyMqbxi/000/gitlab/han12/GEOSX/integratedTests/update/run/thermalMultiphaseFlow/co2_thermal_2d_04 --suppress-move-logging
::::::::::::::::::::
::::::::::::::::::::
/usr/tce/packages/python/python-2.7.16/bin/python -m mpi4py /usr/WS1/han12/.jacamar-ci/builds/4wyMqbxi/000/gitlab/han12/GEOSX/integratedTests/geosxats/helpers/restartcheck.py -a 1e-06 -r 1e-05 -w /usr/WS1/han12/.jacamar-ci/builds/4wyMqbxi/000/gitlab/han12/GEOSX/integratedTests/update/run/thermalMultiphaseFlow/co2_thermal_2d_04/9to11_restart_[0-9]+\.root /usr/WS1/han12/.jacamar-ci/builds/4wyMqbxi/000/gitlab/han12/GEOSX/integratedTests/update/run/thermalMultiphaseFlow/baselines/co2_thermal_2d_04/0to11_restart_[0-9]+\.root
::::::::::::::::::::
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41428]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41470]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41508]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41528]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41548]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41566]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41592]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41628]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41638]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41664]: Insane message length
2: /usr/WS1/han12/.jacamar-ci/builds/4wyMqbxi/000/gitlab/han12/GEOSX/integratedTests/geosxats/helpers/restartcheck.py:246: RuntimeWarning: invalid value encountered in divide
2:   relative_difference = difference / abs_base_arr
0: /usr/WS1/han12/.jacamar-ci/builds/4wyMqbxi/000/gitlab/han12/GEOSX/integratedTests/geosxats/helpers/restartcheck.py:246: RuntimeWarning: invalid value encountered in divide
0:   relative_difference = difference / abs_base_arr
1: /usr/WS1/han12/.jacamar-ci/builds/4wyMqbxi/000/gitlab/han12/GEOSX/integratedTests/geosxats/helpers/restartcheck.py:246: RuntimeWarning: invalid value encountered in divide
1:   relative_difference = difference / abs_base_arr
3: /usr/WS1/han12/.jacamar-ci/builds/4wyMqbxi/000/gitlab/han12/GEOSX/integratedTests/geosxats/helpers/restartcheck.py:246: RuntimeWarning: invalid value encountered in divide
3:   relative_difference = difference / abs_base_arr
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41686]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41706]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41720]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41742]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41770]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41810]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41822]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41840]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41856]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41862]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41892]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41908]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41930]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41964]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:41986]: Insane message length
srun: error: eio_message_socket_accept: slurm_receive_msg[127.0.0.1:42002]: Insane message length
